### DISCUSSION TITLE
Enable preview feature outside of schema

### DISCUSSION THREAD

**Discussion by Gregoor** [prisma/prisma#8671] [Q&A] [answered]
Hi there!

We are using prisma's CLI to generate our Schema, but also want to query the schema based on a `json` field. [That happens to be a preview feature currently](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field). Now the only way to activate those is apparently by editing the schema file. But like I mentioned initially, that one gets auto-generated for us.

Now my question: is there a way to enable preview feature usage somewhere else, e.g. through the PrismaClient?

**Comment by Gregoor** [comment, marked as answer]
Nevermind, I just realized that Prisma's introspect actually does incremental updating and not full-on schema replacing after all. Nifty!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
